### ISSUE TITLE
The source directory does not appear to contain CMakeLists.txt.

### ISSUE THREAD

**Issue by piercus** [UrielCh/opencv4nodejs#75]
Related to #73 

## Scenario

I'm having a simple project with an "opencv4nodejs" key in package.json

```
  "opencv4nodejs": {
    "autoBuildFlags": "-DCMAKE_BUILD_TYPE=RELEASE -DBUILD_opencv_core=ON -DBUILD_opencv_calib3d=ON -DBUILD_opencv_python_tests=OFF -DBUILD_opencv_imgproc=OFF -DBUILD_opencv_imgcodecs=OFF -DBUILD_opencv_videoio=OFF -DBUILD_opencv_highgui=OFF -DBUILD_opencv_video=OFF -DBUILD_opencv_features2d=OFF -DBUILD_opencv_objdetect=OFF -DBUILD_opencv_dnn=OFF -DBUILD_opencv_ml=OFF -DBUILD_opencv_flann=OFF -DBUILD_opencv_photo=OFF -DBUILD_opencv_gapi=OFF -DBUILD_opencv_face=OFF -DBUILD_opencv_img_hash=OFF -DBUILD_opencv_text=OFF -DBUILD_opencv_tracking=OFF -DBUILD_opencv_videostab=OFF  -DBUILD_opencv_xfeatures2d=OFF -DBUILD_opencv_ximgproc=OFF -DWITH_IPP=OFF -DWITH_V4L=OFF -DWITH_FFMPEG=OFF -DWITH_GSTREAMER=OFF -DWITH_GTK=OFF -DWITH_PROTOBUF=OFF -DBUILD_PROTOBUF=OFF -DOPENCV_DNN_OPENCL=OFF -DWITH_WIN32UI=OFF -DWITH_PNG=OFF -DWITH_JPEG=OFF -DWITH_TIFF=OFF -DWITH_WEBP=OFF -DWITH_OPENJPEG=OFF -DWITH_JASPER=OFF -DWITH_OPENEXR=OFF -DBUILD_opencv_python=OFF -DBUILD_opencv_java=OFF"
  },
```

I'm doing 

```
> npm install

added 112 packages, and audited 113 packages in 8s

8 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm timing command:install Completed in 7747ms
npm notice 
npm notice New major version of npm available! 8.19.3 -> 9.5.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.5.0>
npm notice Run `npm install -g npm@9.5.0` to update!
npm notice 
npm verb exit 0
npm timing npm Completed in 7839ms
npm info ok 
```

The auto build seems to not be done so i'm re-running it using 

```
npx build-opencv rebuild
```

### Actual

I've got error : `The source directory "/opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build" does not appear to contain CMakeLists.txt.`

Full logs 
```

info config found opencv4nodejs section in /opt/nodejs/package.json
info init using openCV verison 4.6.0
info init INIT_CWD is defined overwriting root path to /opt/nodejs
info applyEnvsFromPackageJson the following opencv4nodejs environment variables are set in the package.json:
info applyEnvsFromPackageJson autoBuildFlags: -DCMAKE_BUILD_TYPE=RELEASE -DBUILD_opencv_core=ON -DBUILD_opencv_calib3d=ON -DBUILD_opencv_python_tests=OFF -DBUILD_opencv_core=OFF -DBUILD_opencv_imgproc=OFF -DBUILD_opencv_imgcodecs=OFF -DBUILD_opencv_videoio=OFF -DBUILD_opencv_highgui=OFF -DBUILD_opencv_video=OFF -DBUILD_opencv_calib3d=OFF -DBUILD_opencv_features2d=OFF -DBUILD_opencv_objdetect=OFF -DBUILD_opencv_dnn=OFF -DBUILD_opencv_ml=OFF -DBUILD_opencv_flann=OFF -DBUILD_opencv_photo=OFF -DBUILD_opencv_gapi=OFF -DBUILD_opencv_face=OFF -DBUILD_opencv_img_hash=OFF -DBUILD_opencv_text=OFF -DBUILD_opencv_tracking=OFF -DBUILD_opencv_videostab=OFF  -DBUILD_opencv_xfeatures2d=OFF -DBUILD_opencv_ximgproc=OFF -DWITH_IPP=OFF -DWITH_V4L=OFF -DWITH_FFMPEG=OFF -DWITH_GSTREAMER=OFF -DWITH_GTK=OFF -DWITH_PROTOBUF=OFF -DBUILD_PROTOBUF=OFF -DOPENCV_DNN_OPENCL=OFF -DWITH_WIN32UI=OFF -DWITH_PNG=OFF -DWITH_JPEG=OFF -DWITH_TIFF=OFF -DWITH_WEBP=OFF -DWITH_OPENJPEG=OFF -DWITH_JASPER=OFF -DWITH_OPENEXR=OFF -DBUILD_opencv_python=OFF -DBUILD_opencv_java=OFF
info config adding cmake flag "-DCMAKE_BUILD_TYPE=RELEASE"
info config cmake flag "-DBUILD_opencv_core=ON" had no effect.
info config cmake flag "-DBUILD_opencv_calib3d=ON" had no effect.
info config replacing cmake flag "-DBUILD_opencv_python_tests=ON" by "-DBUILD_opencv_python_tests=OFF"
info config replacing cmake flag "-DBUILD_opencv_core=ON" by "-DBUILD_opencv_core=OFF"
info config replacing cmake flag "-DBUILD_opencv_imgproc=ON" by "-DBUILD_opencv_imgproc=OFF"
info config replacing cmake flag "-DBUILD_opencv_imgcodecs=ON" by "-DBUILD_opencv_imgcodecs=OFF"
info config replacing cmake flag "-DBUILD_opencv_videoio=ON" by "-DBUILD_opencv_videoio=OFF"
info config replacing cmake flag "-DBUILD_opencv_highgui=ON" by "-DBUILD_opencv_highgui=OFF"
info config replacing cmake flag "-DBUILD_opencv_video=ON" by "-DBUILD_opencv_video=OFF"
info config replacing cmake flag "-DBUILD_opencv_calib3d=ON" by "-DBUILD_opencv_calib3d=OFF"
info config replacing cmake flag "-DBUILD_opencv_features2d=ON" by "-DBUILD_opencv_features2d=OFF"
info config replacing cmake flag "-DBUILD_opencv_objdetect=ON" by "-DBUILD_opencv_objdetect=OFF"
info config replacing cmake flag "-DBUILD_opencv_dnn=ON" by "-DBUILD_opencv_dnn=OFF"
info config replacing cmake flag "-DBUILD_opencv_ml=ON" by "-DBUILD_opencv_ml=OFF"
info config replacing cmake flag "-DBUILD_opencv_flann=ON" by "-DBUILD_opencv_flann=OFF"
info config replacing cmake flag "-DBUILD_opencv_photo=ON" by "-DBUILD_opencv_photo=OFF"
info config replacing cmake flag "-DBUILD_opencv_gapi=ON" by "-DBUILD_opencv_gapi=OFF"
info config replacing cmake flag "-DBUILD_opencv_face=ON" by "-DBUILD_opencv_face=OFF"
info config replacing cmake flag "-DBUILD_opencv_img_hash=ON" by "-DBUILD_opencv_img_hash=OFF"
info config replacing cmake flag "-DBUILD_opencv_text=ON" by "-DBUILD_opencv_text=OFF"
info config replacing cmake flag "-DBUILD_opencv_tracking=ON" by "-DBUILD_opencv_tracking=OFF"
info config replacing cmake flag "-DBUILD_opencv_videostab=ON" by "-DBUILD_opencv_videostab=OFF"
info config replacing cmake flag "-DBUILD_opencv_xfeatures2d=ON" by "-DBUILD_opencv_xfeatures2d=OFF"
info config replacing cmake flag "-DBUILD_opencv_ximgproc=ON" by "-DBUILD_opencv_ximgproc=OFF"
info config adding cmake flag "-DWITH_IPP=OFF"
info config adding cmake flag "-DWITH_V4L=OFF"
info config adding cmake flag "-DWITH_FFMPEG=OFF"
info config adding cmake flag "-DWITH_GSTREAMER=OFF"
info config adding cmake flag "-DWITH_GTK=OFF"
info config adding cmake flag "-DWITH_PROTOBUF=OFF"
info config adding cmake flag "-DBUILD_PROTOBUF=OFF"
info config adding cmake flag "-DOPENCV_DNN_OPENCL=OFF"
info config adding cmake flag "-DWITH_WIN32UI=OFF"
info config adding cmake flag "-DWITH_PNG=OFF"
info config adding cmake flag "-DWITH_JPEG=OFF"
info config adding cmake flag "-DWITH_TIFF=OFF"
info config adding cmake flag "-DWITH_WEBP=OFF"
info config adding cmake flag "-DWITH_OPENJPEG=OFF"
info config adding cmake flag "-DWITH_JASPER=OFF"
info config adding cmake flag "-DWITH_OPENEXR=OFF"
info config adding cmake flag "-DBUILD_opencv_python=OFF"
info config adding cmake flag "-DBUILD_opencv_java=OFF"
info init Workdir will be: /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53
info install Using openCV 4.6.0
info config adding cmake flag "-DCMAKE_BUILD_TYPE=RELEASE"
info config cmake flag "-DBUILD_opencv_core=ON" had no effect.
info config cmake flag "-DBUILD_opencv_calib3d=ON" had no effect.
info config replacing cmake flag "-DBUILD_opencv_python_tests=ON" by "-DBUILD_opencv_python_tests=OFF"
.... 
info config replacing cmake flag "-DBUILD_opencv_ximgproc=ON" by "-DBUILD_opencv_ximgproc=OFF"
info config adding cmake flag "-DWITH_IPP=OFF"
info config adding cmake flag "-DWITH_V4L=OFF"
info config adding cmake flag "-DWITH_FFMPEG=OFF"
info config adding cmake flag "-DWITH_GSTREAMER=OFF"
info config adding cmake flag "-DWITH_GTK=OFF"
info config adding cmake flag "-DWITH_PROTOBUF=OFF"
info config adding cmake flag "-DBUILD_PROTOBUF=OFF"
info config adding cmake flag "-DOPENCV_DNN_OPENCL=OFF"
info config adding cmake flag "-DWITH_WIN32UI=OFF"
info config adding cmake flag "-DWITH_PNG=OFF"
info config adding cmake flag "-DWITH_JPEG=OFF"
info config adding cmake flag "-DWITH_TIFF=OFF"
info config adding cmake flag "-DWITH_WEBP=OFF"
info config adding cmake flag "-DWITH_OPENJPEG=OFF"
info config adding cmake flag "-DWITH_JASPER=OFF"
info config adding cmake flag "-DWITH_OPENEXR=OFF"
info config adding cmake flag "-DBUILD_opencv_python=OFF"
info config adding cmake flag "-DBUILD_opencv_java=OFF"
info config adding cmake flag "-DCMAKE_BUILD_TYPE=RELEASE"
info config cmake flag "-DBUILD_opencv_core=ON" had no effect.
info config cmake flag "-DBUILD_opencv_calib3d=ON" had no effect.
info config replacing cmake flag "-DBUILD_opencv_python_tests=ON" by "-DBUILD_opencv_python_tests=OFF"
info config replacing cmake flag "-DBUILD_opencv_core=ON" by "-DBUILD_opencv_core=OFF"
info config replacing cmake flag "-DBUILD_opencv_imgproc=ON" by "-DBUILD_opencv_imgproc=OFF"
info config replacing cmake flag "-DBUILD_opencv_imgcodecs=ON" by "-DBUILD_opencv_imgcodecs=OFF"
info config replacing cmake flag "-DBUILD_opencv_videoio=ON" by "-DBUILD_opencv_videoio=OFF"
info config replacing cmake flag "-DBUILD_opencv_highgui=ON" by "-DBUILD_opencv_highgui=OFF"
info config replacing cmake flag "-DBUILD_opencv_video=ON" by "-DBUILD_opencv_video=OFF"
info config replacing cmake flag "-DBUILD_opencv_calib3d=ON" by "-DBUILD_opencv_calib3d=OFF"
info config replacing cmake flag "-DBUILD_opencv_features2d=ON" by "-DBUILD_opencv_features2d=OFF"
info config replacing cmake flag "-DBUILD_opencv_objdetect=ON" by "-DBUILD_opencv_objdetect=OFF"
info config replacing cmake flag "-DBUILD_opencv_dnn=ON" by "-DBUILD_opencv_dnn=OFF"
info config replacing cmake flag "-DBUILD_opencv_ml=ON" by "-DBUILD_opencv_ml=OFF"
info config replacing cmake flag "-DBUILD_opencv_flann=ON" by "-DBUILD_opencv_flann=OFF"
info config replacing cmake flag "-DBUILD_opencv_photo=ON" by "-DBUILD_opencv_photo=OFF"
info config replacing cmake flag "-DBUILD_opencv_gapi=ON" by "-DBUILD_opencv_gapi=OFF"
info config replacing cmake flag "-DBUILD_opencv_face=ON" by "-DBUILD_opencv_face=OFF"
info config replacing cmake flag "-DBUILD_opencv_img_hash=ON" by "-DBUILD_opencv_img_hash=OFF"
info config replacing cmake flag "-DBUILD_opencv_text=ON" by "-DBUILD_opencv_text=OFF"
info config replacing cmake flag "-DBUILD_opencv_tracking=ON" by "-DBUILD_opencv_tracking=OFF"
info config replacing cmake flag "-DBUILD_opencv_videostab=ON" by "-DBUILD_opencv_videostab=OFF"
info config replacing cmake flag "-DBUILD_opencv_xfeatures2d=ON" by "-DBUILD_opencv_xfeatures2d=OFF"
info config replacing cmake flag "-DBUILD_opencv_ximgproc=ON" by "-DBUILD_opencv_ximgproc=OFF"
info config adding cmake flag "-DWITH_IPP=OFF"
info config adding cmake flag "-DWITH_V4L=OFF"
info config adding cmake flag "-DWITH_FFMPEG=OFF"
info config adding cmake flag "-DWITH_GSTREAMER=OFF"
info config adding cmake flag "-DWITH_GTK=OFF"
info config adding cmake flag "-DWITH_PROTOBUF=OFF"
info config adding cmake flag "-DBUILD_PROTOBUF=OFF"
info config adding cmake flag "-DOPENCV_DNN_OPENCL=OFF"
info config adding cmake flag "-DWITH_WIN32UI=OFF"
info config adding cmake flag "-DWITH_PNG=OFF"
info config adding cmake flag "-DWITH_JPEG=OFF"
info config adding cmake flag "-DWITH_TIFF=OFF"
info config adding cmake flag "-DWITH_WEBP=OFF"
info config adding cmake flag "-DWITH_OPENJPEG=OFF"
info config adding cmake flag "-DWITH_JASPER=OFF"
info config adding cmake flag "-DWITH_OPENEXR=OFF"
info config adding cmake flag "-DBUILD_opencv_python=OFF"
info config adding cmake flag "-DBUILD_opencv_java=OFF"
CMake Error: The source directory "/opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
ERR! build Compilation failed, previous calls:
ERR! build rimraf /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/opencv /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/opencv_contrib
ERR! build mkdir -p /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build
ERR! build cd /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53
ERR! build git clone --quiet -b 4.6.0 --single-branch --depth 1 --progress https://github.com/opencv/opencv_contrib.git /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/opencv_contrib
ERR! build git clone --quiet -b 4.6.0 --single-branch --depth 1 --progress https://github.com/opencv/opencv.git /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/opencv
ERR! build export OPENCV_BIN_DIR=/opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build/bin
ERR! build export OPENCV_INCLUDE_DIR=
ERR! build export OPENCV_LIB_DIR=/opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build/lib
ERR! build cd /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build
ERR! build cmake /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/opencv -DCMAKE_INSTALL_PREFIX=/opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build -DCMAKE_BUILD_TYPE=Release -DCMAKE_BUILD_TYPES=Release -DBUILD_EXAMPLES=OFF -DBUILD_DOCS=OFF -DBUILD_TESTS=OFF -DBUILD_opencv_dnn=ON -DENABLE_FAST_MATH=ON -DBUILD_PERF_TESTS=OFF -DBUILD_JAVA=OFF -DBUILD_ZLIB=OFF -DCUDA_NVCC_FLAGS=--expt-relaxed-constexpr -DWITH_VTK=OFF -DOPENCV_ENABLE_NONFREE=ON -DOPENCV_EXTRA_MODULES_PATH=/opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/opencv_contrib/modules -DBUILD_opencv_apps=OFF -DBUILD_opencv_aruco=OFF -DBUILD_opencv_bgsegm=OFF -DBUILD_opencv_bindings_generator=OFF -DBUILD_opencv_bioinspired=OFF -DBUILD_opencv_calib3d=OFF -DBUILD_opencv_ccalib=OFF -DBUILD_opencv_core=OFF -DBUILD_opencv_datasets=OFF -DBUILD_opencv_dnn=OFF -DBUILD_opencv_dnn_objdetect=OFF -DBUILD_opencv_dpm=OFF -DBUILD_opencv_face=OFF -DBUILD_opencv_features2d=OFF -DBUILD_opencv_flann=OFF -DBUILD_opencv_fuzzy=OFF -DBUILD_opencv_gapi=OFF -DBUILD_opencv_hfs=OFF -DBUILD_opencv_highgui=OFF -DBUILD_opencv_img_hash=OFF -DBUILD_opencv_imgcodecs=OFF -DBUILD_opencv_imgproc=OFF -DBUILD_opencv_java_bindings_generator=OFF -DBUILD_opencv_js=OFF -DBUILD_opencv_line_descriptor=OFF -DBUILD_opencv_ml=OFF -DBUILD_opencv_objc_bindings_generator=OFF -DBUILD_opencv_objdetect=OFF -DBUILD_opencv_optflow=OFF -DBUILD_opencv_phase_unwrapping=OFF -DBUILD_opencv_photo=OFF -DBUILD_opencv_python3=OFF -DBUILD_opencv_python_bindings_generator=OFF -DBUILD_opencv_python_tests=OFF -DBUILD_opencv_reg=OFF -DBUILD_opencv_rgbd=OFF -DBUILD_opencv_saliency=OFF -DBUILD_opencv_shape=OFF -DBUILD_opencv_stereo=OFF -DBUILD_opencv_stitching=OFF -DBUILD_opencv_structured_light=OFF -DBUILD_opencv_superres=OFF -DBUILD_opencv_surface_matching=OFF -DBUILD_opencv_text=OFF -DBUILD_opencv_tracking=OFF -DBUILD_opencv_ts=OFF -DBUILD_opencv_video=OFF -DBUILD_opencv_videoio=OFF -DBUILD_opencv_videostab=OFF -DBUILD_opencv_wechat_qrcode=OFF -DBUILD_opencv_world=OFF -DBUILD_opencv_xfeatures2d=OFF -DBUILD_opencv_ximgproc=OFF -DBUILD_opencv_xobjdetect=OFF -DBUILD_opencv_xphoto=OFF 
ERR! install running: cmake /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/opencv -DCMAKE_INSTALL_PREFIX=/opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build -DCMAKE_BUILD_TYPE=Release -DCMAKE_BUILD_TYPES=Release -DBUILD_EXAMPLES=OFF -DBUILD_DOCS=OFF -DBUILD_TESTS=OFF -DBUILD_opencv_dnn=ON -DENABLE_FAST_MATH=ON -DBUILD_PERF_TESTS=OFF -DBUILD_JAVA=OFF -DBUILD_ZLIB=OFF -DCUDA_NVCC_FLAGS=--expt-relaxed-constexpr -DWITH_VTK=OFF -DOPENCV_ENABLE_NONFREE=ON -DOPENCV_EXTRA_MODULES_PATH=/opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/opencv_contrib/modules -DBUILD_opencv_apps=OFF -DBUILD_opencv_aruco=OFF -DBUILD_opencv_bgsegm=OFF -DBUILD_opencv_bindings_generator=OFF -DBUILD_opencv_bioinspired=OFF -DBUILD_opencv_calib3d=OFF -DBUILD_opencv_ccalib=OFF -DBUILD_opencv_core=OFF -DBUILD_opencv_datasets=OFF -DBUILD_opencv_dnn=OFF -DBUILD_opencv_dnn_objdetect=OFF -DBUILD_opencv_dpm=OFF -DBUILD_opencv_face=OFF -DBUILD_opencv_features2d=OFF -DBUILD_opencv_flann=OFF -DBUILD_opencv_fuzzy=OFF -DBUILD_opencv_gapi=OFF -DBUILD_opencv_hfs=OFF -DBUILD_opencv_highgui=OFF -DBUILD_opencv_img_hash=OFF -DBUILD_opencv_imgcodecs=OFF -DBUILD_opencv_imgproc=OFF -DBUILD_opencv_java_bindings_generator=OFF -DBUILD_opencv_js=OFF -DBUILD_opencv_line_descriptor=OFF -DBUILD_opencv_ml=OFF -DBUILD_opencv_objc_bindings_generator=OFF -DBUILD_opencv_objdetect=OFF -DBUILD_opencv_optflow=OFF -DBUILD_opencv_phase_unwrapping=OFF -DBUILD_opencv_photo=OFF -DBUILD_opencv_python3=OFF -DBUILD_opencv_python_bindings_generator=OFF -DBUILD_opencv_python_tests=OFF -DBUILD_opencv_reg=OFF -DBUILD_opencv_rgbd=OFF -DBUILD_opencv_saliency=OFF -DBUILD_opencv_shape=OFF -DBUILD_opencv_stereo=OFF -DBUILD_opencv_stitching=OFF -DBUILD_opencv_structured_light=OFF -DBUILD_opencv_superres=OFF -DBUILD_opencv_surface_matching=OFF -DBUILD_opencv_text=OFF -DBUILD_opencv_tracking=OFF -DBUILD_opencv_ts=OFF -DBUILD_opencv_video=OFF -DBUILD_opencv_videoio=OFF -DBUILD_opencv_videostab=OFF -DBUILD_opencv_wechat_qrcode=OFF -DBUILD_opencv_world=OFF -DBUILD_opencv_xfeatures2d=OFF -DBUILD_opencv_ximgproc=OFF -DBUILD_opencv_xobjdetect=OFF -DBUILD_opencv_xphoto=OFF 
ERR! install in /opt/nodejs/node_modules/@u4/opencv-build/dist/opencv-4.6.0-98f53/build exited with code 1 (for more info, set '--loglevel silly')'
```

### Expected

I would expect opencv to be built

**Comment by piercus** [comment]
Should be fixed by https://github.com/UrielCh/npm-opencv-build/pull/6

**Comment by piercus** [comment]
fixed in v6.4.3 thanks @UrielCh
